src/components/SearchModal/SearchModal.js

```diff
@@ -102,7 +102,8 @@ const propTypes = {
   }).isRequired,
 };
 
-const PAGE_SIZE = 25;
+const PACKAGE_PAGE_SIZE = 25;
+const TITLES_PAGE_SIZE = 100000; // to hide focus after clicking Next or Previous button
```

Comment:
Please also describe that it's because focus index calculations fail with irregular page size